frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -61,43 +62,34 @@ const columns = [
 
 const childColumns = [
   {
-    title: "Permit No.",
-    dataIndex: "permitNo",
-    key: "childPermitNo",
-    render: (text) => <div title="Permit No.">{text}</div>,
+    title: "",
+    dataIndex: "key",
+    key: "Key",
```

Comment:
Any reason why this isn't something like `amendmentNo`?

Reply:
No, I'm not sure why I went with key. Let's change it to amendmentNumber which is what I pass into it.